strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/NugetArtifactCoordinates.java

```diff
@@ -20,9 +21,9 @@
 @XmlAccessorType(XmlAccessType.NONE)
 public class NugetArtifactCoordinates extends AbstractArtifactCoordinates<NugetArtifactCoordinates, Version>
 {
-    public static final String ID = "id";
-    public static final String VERSION = "version";
-    public static final String EXTENSION = "extension";
+    public static final String ID = "Id";
```

Comment:
Just out of curiousity (and for my better understanding):
* Is NuGet case-sensitive?
* Why do we need this change? (It's lowercase for all of the other artifact coordinates implementations).

Reply:
yes, nuget have it with a capital letter
so I thought that we should have with original case

Reply:
> I would disagree, as it would be inconsistent compared compared to the artifact coordinates for the other layout providers

what you mean? I think we shouldn't compare coordinates from different layout providers at all

Reply:
@sbespalov : 

* I think that all of the artifact coordinates should be in lowercase for every layout provider, so that we could have consistency.
* For NuGet's queries over REST we can have those with as they are in NuGet/NuGetGallery.

Reply:
@carlspring maybe you are right, let's leave this in lowercase for now